### ISSUE TITLE
Outline.feature is not giving any result. How are you specifying the data in examples

### ISSUE THREAD

**Issue by rkbala** [grasshopper7/testcukefeatproc#2]

**Comment by rkbala** [comment]
Outline.feature is not giving any result. How are you specifying the test data in examples? @grasshopper7

**Comment by grasshopper7** [comment]
The data is defined in these folders - https://github.com/grasshopper7/testcukefeatproc/tree/master/testcukefeatproc/src/test/resources/data

**Comment by rkbala** [comment]
Yes I could see all those details. The question is how your are specifying excel test data in “examples” of scenario outline? I see you have given the syntax|Data:|excelpath|sheet name|cell value. But what would be the actual values in this? Can you please give an example? Should It be mentioned like |Data: This|excelpath|sheet name|cell?

**Comment by grasshopper7** [comment]
| Data: | src/test/resources/data/excel/exceltwo.xlsx | Sheet1 | A1:C4 |
Check the template folder - https://github.com/grasshopper7/testcukefeatproc/blob/master/testcukefeatproc/src/test/resources/template/outline/outline.feature

**Comment by rkbala** [comment]
Yea I saw that already. But it is not working. Outline.feature is not executing any steps. Doesn’t that first row in example will be considered as column headers in scenario outline?

**Comment by grasshopper7** [comment]
Just tried it. it works. The first line is the header. How are u running it? U need to use the maven lifecycle for this to work. try 'clean install'. The feature file should be generated in the 'src\test\resources\feature' folder. This configuration can be set here https://github.com/grasshopper7/testcukefeatproc/blob/6f0bb1841693718c1200545383126288767ef93b/testcukefeatproc/pom.xml#L56

**Comment by rkbala** [comment]
I downloaded your repo and running with maven only. I have not changed anything in your project. 

I have changed config like below:
							<arguments>

								<argument>src/test/resources/feature/</argument>
							</arguments>

**Comment by rkbala** [comment]
mvn clean install worked. I was under impression that itemplate/feature files will get the data from excel and send it to step defn. But here, you are creating new feature files with all data tables extracted from excel. 

Do you know a way to load excel and convert it to cucumber datatable, so that we can use the required excel cell in step definitions?

**Comment by grasshopper7** [comment]
Sorry it fell between the cracks. It could be done by extending the code here. Will look into this but cannot guarantee any timelines
